<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="atualiza_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>224</y>
     <width>176</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Atualizar dados cadastrais</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_matricular">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>288</y>
     <width>129</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Matricular disciplina</string>
   </property>
  </widget>
  <widget class="QPushButton" name="atualizar_senha_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>256</y>
     <width>106</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Atualizar senha</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="bem_vindo_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>16</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>171</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_1"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>171</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_2"/>
  </widget>
  <widget class="QLabel" name="bem_vindo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
